src/main/java/com/cloudogu/smeagol/wiki/infrastructure/PageController.java

```diff
@@ -122,6 +141,10 @@ private Path getMoveTo() {
             }
             return Path.valueOf(moveTo);
         }
+        private CommitId getRestore() {
+            if (Strings.isNullOrEmpty(restore)) return null;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS00122)